task/04-arrays-tasks.js

```diff
@@ -102,7 +102,7 @@ function getArrayOfStrings(arr) {
  *    [ false, 0, NaN, '', undefined ]   => [ ]
  */
 function removeFalsyValues(arr) {
-   throw new Error('Not implemented');
+    return arr.filter(v => Boolean(v) == true);
```

Comment:
This line can be simplified:

```suggestion
    return arr.filter(v => v);
```